Seminar 4/Seminar 4 HW/task_3.py

```diff
@@ -0,0 +1,14 @@
+# Напишите функцию принимающую на вход только ключевые параметры и возвращающую словарь,
+# где ключ — значение переданного аргумента, а значение — имя аргумента.
+# Если ключ не хешируем, используйте его строковое представление.
+def params_to_dict(**kwargs):
+    res = {}
+    for k, v in kwargs.items():
+        if v.__hash__:
+            res[v] = k
+        else:
+            res[v.__str__()] = k
```

Comment:
Либо можно чуть упростить.
```suggestion
        if not v.__hash__:
            v = str(v)

        res[v] = k
```